crates/vhost/Cargo.toml

```diff
@@ -24,13 +24,13 @@ vhost-user-master = ["vhost-user"]
 vhost-user-slave = ["vhost-user"]
 
 [dependencies]
-bitflags = "1.0"
-libc = "0.2.39"
+bitflags = "1.3.2"
```

Comment:
Same here. For non-zero versions, SemVer allows the `bitflags` dependency by any version that's equal or bigger than `1.0` and smaller than `2.0`.

Reply:
Ditto, this isn't in the updated version.